src/test/java/dev/blaauwendraad/masker/randomgen/InvalidJsonPrettyPrinter.java

```diff
@@ -0,0 +1,115 @@
+package dev.blaauwendraad.masker.randomgen;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.PrettyPrinter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static dev.blaauwendraad.masker.json.util.AsciiCharacter.CARRIAGE_RETURN;
+import static dev.blaauwendraad.masker.json.util.AsciiCharacter.HORIZONTAL_TAB;
+import static dev.blaauwendraad.masker.json.util.AsciiCharacter.LINE_FEED;
+import static dev.blaauwendraad.masker.json.util.AsciiCharacter.SPACE;
+
+/**
+ * This class is DefaultPrettyPrinter with additional logic to add white spaces to JSON
+ */
+public class InvalidJsonPrettyPrinter implements PrettyPrinter {
+
+    private static final List<Character> JSON_CONTROL_CHARACTERS = List.of(
+            '{',
+            '}',
+            '[',
+            ']',
+            ':',
+            '\'',
+            '"',
+            ',',
+            'a',
+            't',
+            'f',
+            'n'
```

Comment:
Probably we should also add numeric characters (-, e, E, 0, 1) but can also do it in later PR